tests/unit/scatter-test.js

```diff
@@ -105,3 +106,18 @@ test('Test domain without domain padding', function(assert) {
   assert.equal(component.get('xScale')(0.017), 497.65);
   assert.equal(component.get('yScale')(0.03), 193.815);
 });
+
+test('Test legend color override', function(assert) {
+  var component = this.subject(scatterContent);
```

Comment:
I think we do, don't we? We test that the color is correct both before overriding and after overriding, and the "before overriding" case is the same as the "don't override at all" case.

Or was that part of the test added after your comment?